projects/varnish/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eu
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+
+git apply --ignore-whitespace $SRC/patch.diff
+# build project
+./autogen.sh
+./configure
+make -j$(nproc)
+cp bin/varnishd/esi_parse_fuzzer $OUT/
```

Comment:
Should we limit ourselves to the ESI fuzzing or work on having support for more kind of fuzzing directly in tree?

I'm not familiar with how oss-fuzz operates but it looks like we shouldn't expect much value from just fuzzing ESI so we should probably consider a greater scope even though that is currently not available directly in our master branch.

Reply:
> Should we limit ourselves to the ESI fuzzing or work on having support for more kind of fuzzing directly in tree?

More fuzzers are definitely welcome

Should this line become `cp bin/varnishd/*_fuzzer $OUT/` so that other fuzzers will work ?

Reply:
+1

Reply:
pushed